server/lib/queries.js

```diff
@@ -36,7 +36,7 @@ const getTotalDonations = () => {
     SELECT SUM(${generateFXConversionSQL()}) AS "totalDonationsInUSD"
     FROM "Transactions"
     WHERE amount > 0 AND "PaymentMethodId" IS NOT NULL
-  `.replace(/\n/g, ' '), // this is to remove the new lines and save log space.
+  `.replace(/\s\s+/g, ' '), // this is to remove the new lines and save log space.
```

Comment:
Just \s+ means at least one occurrence or more. So no need for the second one which would mean at least two occurrences or more.